client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/index.js

```diff
@@ -20,77 +20,70 @@ import RatesStep from './rates-step';
-			>
-				<div className="label-purchase-modal__content">
-					<div className="label-purchase-modal__main-section">
-						<AddressStep
-							type="origin"
-							title={ translate( 'Origin address' ) }
-							siteId={ props.siteId }
-							orderId={ props.orderId }
-						/>
-						<AddressStep
-							type="destination"
-							title={ translate( 'Destination address' ) }
-							siteId={ props.siteId }
-							orderId={ props.orderId }
-						/>
-						<PackagesStep siteId={ props.siteId } orderId={ props.orderId } />
-						{ props.isCustomsFormRequired && (
-							<CustomsStep siteId={ props.siteId } orderId={ props.orderId } />
-						) }
-						<RatesStep siteId={ props.siteId } orderId={ props.orderId } />
-					</div>
-					<Sidebar siteId={ props.siteId } orderId={ props.orderId } />
-				</div>
-			</Modal>
-		) : null
-	);
+	return showPurchaseDialog ? (<><Modal
+		className="woocommerce label-purchase-modal wcc-root"
+		shouldCloseOnClickOutside={false}
+		onRequestClose={onClose}
+		title={translate('Create shipping label',
+			'Create shipping labels',
+			{ count: Object.keys(props.form.packages.selected).length }
+		)}
+	>
+		<div className="label-purchase-modal__content">
+			<div className="label-purchase-modal__main-section">
+				<AddressStep
+					type="origin"
+					title={translate('Origin address')}
+					siteId={props.siteId}
+					orderId={props.orderId}
+				/>
+				<AddressStep
+					type="destination"
+					title={translate('Destination address')}
+					siteId={props.siteId}
+					orderId={props.orderId}
+				/>
+				<PackagesStep siteId={props.siteId} orderId={props.orderId}/>
+				{props.isCustomsFormRequired && (<CustomsStep siteId={props.siteId} orderId={props.orderId}/>)}
+				<RatesStep siteId={props.siteId} orderId={props.orderId}/>
+			</div>
+			<Sidebar siteId={props.siteId} orderId={props.orderId}/>
+		</div>
+	</Modal>
+		<FeatureAnnouncement siteId={props.siteId} orderId={props.orderId}/>
+	</>) : null;
 };
 
 LabelPurchaseModal.propTypes = {
-	siteId: PropTypes.number.isRequired,
-	orderId: PropTypes.number.isRequired,
+	siteId: PropTypes.number.isRequired, orderId: PropTypes.number.isRequired,
 };
 
-const mapStateToProps = ( state, { orderId, siteId } ) => {
-	const loaded = isLoaded( state, orderId, siteId );
-	const shippingLabel = getShippingLabel( state, orderId, siteId );
+const mapStateToProps = (state, { orderId, siteId }) => {
+	const loaded = isLoaded(state, orderId, siteId);
+	const shippingLabel = getShippingLabel(state, orderId, siteId);
 	return {
 		loaded,
 		form: loaded && shippingLabel.form,
 		showPurchaseDialog: shippingLabel.showPurchaseDialog,
-		isCustomsFormRequired: isCustomsFormRequired( state, orderId, siteId ),
+		isCustomsFormRequired: isCustomsFormRequired(state, orderId, siteId),
 	};
 };
 
 const mapDispatchToProps = dispatch => {
-	return bindActionCreators( { exitPrintingFlow }, dispatch );
```

Comment:
I was expecting eslint to check my code, but seemingly it doesn't not, I was not sure what's the preferred formatting on this repo, I'd update if I you suggest a reference we follow for this repo.

Reply:
`.eslintignore` ignores this folder so `eslint` won't report any errors. I don't think we need to remove the whitespace/newline because they look like they were adhering to the WordPress/Woo Standard. Removing the space seems to be a step away from the standard, which is why I am calling it out as a code reviewer.

> I was not sure what's the preferred formatting on this repo

I usually default to the WordPress/Woo standards. Ideally, `eslint --fix` will fix this for us. I don't want to come off as being nitpicky and I also don't have a strong argument given that this file is ignored in `.eslintignore`. We can check with the team or you can use your best judgement.

Reply:
> Ideally, eslint --fix will fix this for us.

It doesn't fix the issues, not it does it for the newly added file (client/components/migration/feature-announcement.jsx) 
I remember we had a discussion about this on Feb 22nd 1Ju-42fxF1FiOymv0mrRAnJZfFsWZ3jPTGu06BGKX8Co-gdoc and we agreed to leave these things to our linters!

Reply:
The commit 2e2f50b2daf9899358328ea113ff7764566c46f5  reverts the changes in the old file, but the linter didn't do anything for the new file.